Ports/vitetris/package.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=vitetris
+useconfigure="true"
+version="0.59.1"
+files="https://github.com/vicgeralds/vitetris/archive/refs/tags/v${version}.tar.gz vitetris.tar.gz"
+configopts="--without-xlib --without-joystick --without-network"
+
+configure() 
+{
+    chmod +x "${workdir}"/"$configscript"
+    run ./"$configscript" $configopts
+ }
```

Comment:
```suggestion
configure() {
    run chmod +x "$configscript"
    run ./"$configscript" $configopts
 }
```

I realise this is taken from `.port_include.sh`, but using `run` in both cases should be preferred.

Reply:
Fixed